.github/project.yml

```diff
@@ -1,4 +1,4 @@
 name: Sundrio
 release:
   current-version: 0.200.0
```

Comment:
```suggestion
  current-version: 0.200.1
```

Reply:
Sorry, I've pushed a commit to correct this. After reviewing the release workflow more it looks like the next-version needed corrected as well.